ansible/roles_ocp_workloads/ocp4_workload_bookbag_user/tasks/workload.yaml

```diff
@@ -0,0 +1,72 @@
+---
+
+- name: Get User Data
+  set_fact:
+    _user_data: >-
+      {{ lookup('file', hostvars.localhost.output_dir ~ '/user-data.yaml', errors='ignore')
+      | from_yaml }}
+
+- name: Fallback for User Data
+  when: _user_data | default({}) | length == 0
+  set_fact:
+    _user_data: {}
+
+- name: Get User Info
+  set_fact:
+    _user_info_messages: >-
+      {{ lookup('file', hostvars.localhost.output_dir ~ '/user-info.yaml', errors='ignore')
+      | from_yaml }}
+
+- name: Fallback for User Info
+  when: _user_info_messages | default({}) | length == 0
+  set_fact:
+    _user_info_messages: {}
+
+- name: Set workshop default vars
+  set_fact:
+    _workshop_default_vars:
+      GUID: "{{ guid }}"
+      USERNAME: "{{ ocp_username }}"
+      user_info_messages: "{{ _user_info_messages }}"
```

Comment:
For workshop vars, the user_info_messages list should be joined with newlines to a single string.

Reply:
Done